tests/test_jensen.py

```
@@ -288,6 +288,73 @@ async def test_build_pin_unlock_command_custom(
+        mock_jensen_config_entry,
+    ):
+        """Test coordinator refreshes Jensen PIN before controller command execution."""
+        coordinator = AdjustableBedCoordinator(hass, mock_jensen_config_entry)
+        coordinator._client = MagicMock()
+        coordinator._client.is_connected = True
+        coordinator._controller = MagicMock()
+        coordinator._controller.send_pin = AsyncMock()
+
+        command_called = AsyncMock()
+
+        async def _command_fn(_controller):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_command_fn`

Add return type annotation: `None`

(ANN202)